packages/components/badge/__tests__/badge.test.tsx

```diff
@@ -62,4 +62,15 @@ describe('Badge', () => {
     await nextTick()
     expect(wrapper.find('.el-badge__content').text()).toEqual('80')
   })
+
+  test('showZero', async () => {
+    const badgeValue = ref(0)
+    const wrapper = mount(() => (
+      <Badge showZero={true} value={badgeValue.value} />
+    ))
+    expect(wrapper.find('.el-badge__content').text()).toEqual('0')
+    badgeValue.value = 80
+    await nextTick()
+    expect(wrapper.find('.el-badge__content').text()).toEqual('80')
```

Comment:
When changing `showZero` to false, it's necessary to test the value `0`.

Reply:
Should I add the situation where showZero is false？

Reply:
Yes, set `const showZero= ref(true);` and change the vaule.